tests/runtest.sh

```diff
@@ -437,6 +440,85 @@ function create_core_overlay {
     copy_test_native_bin_to_test_root $coreOverlayDir
 }
 
+function create_testhost
+{
+    if [ ! -d "$testHostDir" ]; then
+        exit_with_error "$errorSource" "Did not find the test host directory: $testHostDir"
+    fi
+
+    # Initialize test variables
+    local buildToolsDir=$coreClrSrc/Tools
+    local dotnetExe=$buildToolsDir/dotnetcli/dotnet
+    local coreClrSrcTestDir=$coreClrSrc/tests
+    
+    if [ -z $coreClrBinDir ]; then
+        local coreClrBinDir=${coreClrSrc}/bin
+        export __CoreFXTestDir=${coreClrSrc}/bin/tests/CoreFX
```

Comment:
Reducing verbosity. Instead of running `runtest.sh` with both `--coreclr-src` and `--coreclr-bin` we infer the bin directory. 
If someone were to supply both it would change the output directory of the helper projects and logs.